package.json

```diff
@@ -45,6 +46,9 @@
     "test": "vitest run --coverage",
     "test:watch": "vitest --coverage"
   },
+  "dependencies": {
+    "chalk": "^5.3.0"
+  },
```

Comment:
Unfortunately, we can't have any non-dep dependencies for this repo, since we'll get flagged if they have any security issues. For this use case, I'd recommend keeping the message as plain text rather than colorifying it. The colors likely won't show up in a CI environment anyway.